master/reference/public-cloud/azure.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Deploying Calico on Azure
+no_canonical: true
+---
+
+{{site.prodname}} in [Microsoft Azure][Azure]{:target="_blank"} supported in [policy-only][PolicyMode] mode.
+Networking part covered by Azure user-defined routes. {{site.prodname}} IPAM needs
+to be configured in host-local mode and used in conjunction with Kubernetes pod CIDR assignments.
+
+#### Routing Traffic
+
+Only available option for traffic routing in Azure with {{site.prodname}} is [Azure user-defined routes][AzureUDR]{:target="_blank"}.
+
+##### Azure user-defined routes
```

Comment:
An alternative to this would be to use flannel networking (a la "canal") since Azure is totally fine with vxlan.